app.go

```diff
@@ -175,12 +175,12 @@ func (a *App) ValidateApiKey(apiKey string) string {
 
 		if err != nil {
 			runtime.LogErrorf(a.ctx, "cannot check WhoAmI: %s", err.Error())
-			return err.Error()
+			return "cannot validate the credentials for the given ApiKey"
```

Comment:
should we be using the same error message here? This is the error to the call to WhoAmI

Reply:
I would not like to display to the user that is returned internally